### ISSUE TITLE
open boundary with loss coefficient using HVAC namelist group

### ISSUE THREAD

**Issue by fdsuser** [firemodels/fds#3209]
I'm trying to simulate a fire with an open boundary at one side. The open boundary requires loss coefficient of 1. I tried two methods:

HVAC_SAMPLE_TEST_1
I set up a HVAC network adjacent to open boundary. I'm getting an numerical instability.

HVAC_SAMPLE_TEST_2
I set up a HVAC network at one side using AMBIENT=.TRUE. 
The simulation shows the air trapped inside and it is not connected to exterior.

Please advise how to make the air flow through open boundary with loss coefficient of 1 at the exit. Thanks in advance. 

[HVAC_SAMPLE_TEST_2.txt](https://github.com/firemodels/fds-smv/files/57501/HVAC_SAMPLE_TEST_2.txt)

[HVAC_SAMPLE_TEST.txt](https://github.com/firemodels/fds-smv/files/57495/HVAC_SAMPLE_TEST.txt)

**Comment by drjfloyd** [comment]
It isn't really clear what you are trying to accomplish.  If you want to have flow out of an open boundary, why not just use an OPEN boundary?  With an HVAC vent you will only get flow in one direction, you will not establish bidirectional flow over the full height of the opening.

With test 1 you have only one live cell between your vent and the open boundary.  This is likely not making life easy for the pressure solver.

In test 2 you haven't assigned any losses. Your HVAC vent is extremely large compared to the overall area of your domain boundaries. You have a 10 kW fire.  It only requires a very tiny outflow to keep up with the pressure rise caused by a 10 kW fire in a 1 m^3 volume.  Look at the hrr.csv file and you will see that pressure is not rising linearly over time as it would if the compartment were sealed. Instead the pressure bounces around zero.

**Comment by fdsuser** [comment]
I'd like to simulate airflow through a boundary leads to exterior, but the boundary would need to have loss coefficent of some number, (i.e. loss= 1,2,3,etc.) representing vent grating or shortened part of long conduit. The flow through exterior would be inflow or outflow, and the direction is unknown as it would depend on the simulated result. It would be bidirectional flow over full length of height.

My questions:
1. If I use the open boundary, how can I prescribe loss coefficient of some number at the open boundary? 
2. Does open boundary have any default loss coeffient attached? (0.5 for entering flow and 1.0 for exiting flow.). If so, can I modify this default value?
3.. Since the flow would be birectional over full height, test 1 would not be the right approach. For Test 2, I was supposed to put "loss=1,1" at the HVAC duct (end of line 22). But I know it wouldn't solve the problem. In fact, the loss coefficient could be 2, 3 or any other number. Is HVAC vent appropriate for the airflow that I'm trying simulate? Is there any other method to achieve the airflow?

Thank you.

**Comment by rmcdermo** [comment]
You cannot specify a loss coefficient for an open bc.

You can put an HVAC bc interior just prior to the end of a duct with an
open bc at the domain boundary.

**Comment by mcgratta** [comment]
You might want to look at the use of Lagrangian particles to form a SCREEN. But be prepared to create a verification case that demonstrates that the screen has the equivalent loss as your duct.

**Comment by fdsuser** [comment]
I'm looking at 14.4.8 Screens on page 163 of FDS 6.3.2 user manual. 
On sample input file:
&SURF ID='SCREEN', THICKNESS=0.00015, GEOMETRY='CYLINDRICAL', 
MATL_ID='ALUMINUM' /

On last paragraph:
"In the example, an aluminum screen with a 40 % free area and an 0.0003 m wire diameter is placed normal to the x-axis."

Please verify the input file for wire diameter is correct. For 0.0003 m wire diameter, do we prescribe THICKNESS 0.00015m in input file?

Thanks.

**Comment by drjfloyd** [comment]
Per the User's Guide for a cylindrical or spherical SURF, the THICKNESS is the radius.

**Comment by fdsuser** [comment]
Thank you. I will approach modeling this vent condition using SCREEN as you suggested.
